packages/npm/@amazeelabs/gatsby-fragments/src/generate.test.ts

```diff
@@ -0,0 +1,38 @@
+import { readFileSync } from 'fs';
+import { sync } from 'glob';
+
+import { generate } from './generate';
+
+describe('generate', () => {
+  const filesAmountEquality = () => sync('./test/*.gql').length === sync('./test/*.fragment.ts').length;
```

Comment:
Running now tests in isolation with mock-fs https://github.com/AmazeeLabs/silverback-mono/commit/c44f3e9eb6210b8eb210789e008fc77a7f5c5204. 
Also added async/await so the test would stay valid if we decide to use `writeFile` and `readFile` in `generate` instead of `readFileSync` and `writeFileSync`.